contexts/credentials/v2

```diff
@@ -43,17 +44,20 @@
     },
     "cnf": {
       "@id": "https://www.iana.org/assignments/jwt#cnf",
-      "@context": {
-        "@protected": true,
-        "kid": {
-          "@id": "https://www.iana.org/assignments/jwt#kid",
-          "@type": "@id"
-        },
-        "jwk": {
-          "@id": "https://www.iana.org/assignments/jwt#jwk",
-          "@type": "@json"
+      "@context": [
+        null,
```

Comment:
can you explain what this is "fixing" ?

Reply:
> can you explain what this is "fixing" ? (in more detail than the PR description has).

You defined `kid` as `https://www.iana.org/assignments/jose#kid"` above... and then re-define it to `https://www.iana.org/assignments/jwt#kid` below -- don't know why you are switching between `.../jose#`  and `.../jwt#`, but presuming you meant to do that, this is ensuring that your re-definition when used in `cnf` works as intended.

If you didn't mean to do that, we're going to have to rework all the JWT/JOSE terms to align them with either `.../jose#`  or `.../jwt#`.

Reply:
So what this is fixing is:

Using the more specific registry to define the terms underneath `cnf`, ensuring that `kid` and `jwk` are defined according to the JWT registry, and not "generic jose".

This seems like an excellent improvement.

Reply:
@OR13,

If anything else needs to be defined under `cnf`, you'll want to do so as well. Right now only `kid` and `jwk` will be defined.

Reply:
@dlongley that seems like breaking changes, `cnf` should get `issuer dependent` terms when a none "registered claim name" is used imo.

Reply:
@OR13,

If you want to keep all of the other terms we can just remove `null` and just let these other specific overrides happen. If you want to clear all the other terms we should keep `null` and add back in whatever else you specifically want.

Reply:
I'm happy to just drop the `null` and let the two overrides take place, leaving the other things there. Seems like the simplest solution to me at this time.